plutus-core/plutus-core.cabal

```diff
@@ -286,7 +286,7 @@ library
     , bytestring
     , bytestring-strict-builder
     , cardano-crypto
-    , cardano-crypto-class        ^>=2.1.2
+    , cardano-crypto-class        ^>= 2.2
```

Comment:
Any reason not to do this? Plutus seems to compile with this change.

Reply:
Reason not to do this is because `cardano-ledger` does not compile with the new version of `cardano-crypto-class`.